src/jit/lsraxarch.cpp

```diff
@@ -2382,6 +2382,13 @@ void LinearScan::BuildHWIntrinsic(GenTreeHWIntrinsic* intrinsicTree)
             break;
         }
 
+        case NI_SSE41_Extract:
+            if (intrinsicTree->gtSIMDBaseType == TYP_FLOAT)
+            {
+                info->internalIntCount += 1;
```

Comment:
This is needed to get the value back in an XMM register, correct?

Reply:
Yes, the original `extractps` instruction writes the result (a floating-point number) into a GPR, but our API returns a `float`. So we need the int-register as the `dst` of `extractps`.